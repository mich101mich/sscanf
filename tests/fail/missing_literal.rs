fn main() {
    sscanf::sscanf!("5");
    sscanf::sscanf!("5",);
    
    let input = "hi";
    sscanf::sscanf!(input, '{', std::vec::Vec<usize>);

    let fmt = "{}";
    sscanf::sscanf!(input, fmt, std::vec::Vec<usize>);
}
