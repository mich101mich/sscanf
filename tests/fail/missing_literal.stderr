error: At least 2 Parameters required: Missing format string
 --> tests/fail/missing_literal.rs:2:5
  |
2 |     sscanf::sscanf!("5");
  |     ^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `sscanf::sscanf` (in Nightly builds, run with -Z macro-backtrace for more info)

error: At least 2 Parameters required: Missing format string
 --> tests/fail/missing_literal.rs:3:24
  |
3 |     sscanf::sscanf!("5",);
  |                        ^

error: expected string literal
 --> tests/fail/missing_literal.rs:6:28
  |
6 |     sscanf::sscanf!(input, '{', std::vec::Vec<usize>);
  |                            ^^^

error: expected string literal
 --> tests/fail/missing_literal.rs:9:28
  |
9 |     sscanf::sscanf!(input, fmt, std::vec::Vec<usize>);
  |                            ^^^
